spec/blast_versions/blast_2.9.0/import_spec_capybara_local_2.9.0.rb

```diff
@@ -467,7 +470,8 @@
     # Click on the first Alignment download button on the page and wait for the
     # download to finish.
 
-    page.execute_script("$('.download-aln:eq(1)').click()")
+    expect(page).to have_selector('.download-aln', minimum: 30) # Wait for react to render all alignments
```

Comment:
FYI: lower in the spec it tries to tick some checkboxes lower on the page and sometimes the test is faster than the react renderer, so adding this clause makes capybara wait until at least the first 30 alignments are rendered, ensuring checkbox selection will not fail

Reply:
great catch